packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -168,17 +126,7 @@ export const PaymentForm = ({
         required
         autoFocus
         spellCheck={false}
-        onValidate={(value, focused) => {
```

Comment:
First refactor: moved the validation from in-line definitions in the form to statically-defined functions in the module.